razorpay-payments.php

```diff
@@ -8,10 +8,14 @@
 Author URI: https://razorpay.com
 */
 
+require_once __DIR__.'/includes/razorpay-webhook.php';
+
 require_once __DIR__.'/razorpay-sdk/Razorpay.php';
 use Razorpay\Api\Api;
 
 add_action('plugins_loaded', 'woocommerce_razorpay_init', 0);
+add_action('admin_post_nopriv_rzp_webhook', 'razorpay_webhook_init'); // second - admin_post_nopriv
```

Comment:
We don't have logged-in-users submitting in this case, it is the razorpay servers